Developer Samples/WindowsContainers/inrule-catalog/readme.md

```diff
@@ -0,0 +1,41 @@
+# README for irCatalog Docker file
+
+## Important notes on building the image
+
+* You must have a valid license for irCatalog.
+* Prior to building the image, copy the irServer RepositoryService IisService assets (default is usually `C:\Program Files (x86)\InRule\irServer\RepositoryService\IisService`) into this repository's [irCatalog directory](/irCatalog/)
```

Comment:
Good catch will resolve in next push

Reply:
@ehertlein -- Q: you got tripped on  needing to copy only the contents of the /IisService folder, do you think there's better copy that could help others avoid that potential pitfall? 

I was thinking of removing the first bullet about licensing since it's already mentioned in the third, then maybe add a bullet about how the catalog service is a WCF website and thus set expectations as to the types of files to expect to copy, or is that obvious from looking at the DOCKERFILE?